Solutions/Marain.UserNotifications.Specs/Features/ManagementApi/GenerateTemplate.feature

```diff
@@ -0,0 +1,29 @@
+﻿@perFeatureContainer
+@useApis
+@useTransientTenant
+Feature: Generate Template
+
+Scenario: Generate populated template
+	Given I have created and stored a notification template
+		| notificationType             | smsTemplate                                         |
+		| marain.notifications.test.v1 | {"body": "A new lead was added by {{leadAddedBy}}"} |
+	And I have created and stored a user preference for a user
+		| userId | email         | phoneNumber | communicationChannelsPerNotificationConfiguration  |
+		| 1      | test@test.com | 041532211   | {"marain.notifications.test.v1": ["email", "sms"]} |
+	When I send the generate template API request
+		"""
```

Comment:
:+1: for multiline strings in .feature files. I have been meaning to look up the syntax for this!

Reply:
Yeah, this was already implemented in this project. You can thank Jon for this 😊